package.json

```diff
@@ -9,7 +9,8 @@
     "nocov": "node ./node_modules/.bin/mocha test/**.test.js",
     "test": "semistandard && npm run coverage",
     "coverage": "node_modules/.bin/istanbul cover node_modules/.bin/_mocha --report lcov -- -R spec",
-    "lint": "semistandard"
+    "lint": "semistandard",
+    "test:integration": "node ./node_modules/.bin/mocha test/index.integration.js"
```

Comment:
FYI - you don't need to prefix `node ./node_modules/.bin/` when defining npm scripts - in fact you *shouldn't*. npm will automatically include binaries from `node_modules/.bin` directories in `$PATH` when executing scripts, so any local dependency which defines a binary (such as mocha in this case) can be executing simply with its command name.

e.g. `"test:integration": "mocha test/integration.js"`

The reasons you shouldn't do this are:

* npm install locations are not guaranteed to be `./node_modules` because npm will de-dupe modules if it already finds them installed further up the file tree.
* binaries might not be `node` executables, they could just as easily be bash scripts or anything else, so defining `node` as an executable is potentially prone to error if the target file is not expecting to execute with `node`. Generally speaking, a defined binary will [use a hashbang to define its own execution command](https://github.com/mochajs/mocha/blob/master/bin/mocha#L1).